packages/web-api/src/WebClient.ts

```diff
@@ -82,7 +78,6 @@ export interface WebAPICallResult {
     // `chat.postMessage` returns an array of error messages (e.g., "messages": ["[ERROR] invalid_keys"])
     messages?: string[];
   };
-  [key: string]: unknown;
```

Comment:
The other key change to address #1323: request responses allow for arbitrary keys, making them type-unsafe.

Reply:
If the currently generated response types miss some of response properties, this change may affect TypeScript users. That being said, we can improve it by providing complete typing as much as possible.